src/pages/_app.tsx

```diff
@@ -11,7 +11,10 @@ function getLibrary(provider) {
   return new ethers.providers.Web3Provider(provider) // this will vary according to whether you use e.g. ethers or web3.js
 }
 
-const Web3ReactProviderDefault = createWeb3ReactRoot(DefaultProviderName)
+const Web3ReactProviderDefault = dynamic(
+  () => import('../components/defaultprovider'),
+  { ssr: false }
```

Comment:
this essentially kills SSG & SSR for the entire site, correct?

The way we're headed with CMS discussion would fundamentally require SSG, so we may need to pull on the thread of serving web3-dehydrated from server site and hydrating web3 thereafter. 

Maybe we return `<SiteLayout web3={false}>` from the server as default and use the prop downstream to disable any web3 components like the nav button or claim page. cc @nateawelch

Reply:
Both SSG & SSR will still be supported with dynamic.

Reply:
Im having a difficult time looking this up actually, ssr: false does disable ssr but I don't know if the nested components inherit this. do you have a way of proving ssr still works through the rest of the app when the parent component is dynamically loaded?

Reply:
I can confirm that .next/server/pages does have the pre rendered html files. but I don't know enough about react to say if that is conclusive.